src/lib/arch/riscv/Instruction_decode.cc

```diff
@@ -19,6 +19,24 @@ namespace riscv {
 Register csRegToRegister(unsigned int reg) {
   // Check from top of the range downwards
 
+  // Metadata could produce either 64-bit floating point register or 32-bit
+  // floating point register. Map both encodings to the same SimEng register.
+  // Only 64-bit registers are supported
+
+  // Modulus ensures only 64 bit registers are recognised
+  if (RISCV_REG_F31_64 >= reg && reg >= RISCV_REG_F0_64 && reg % 2 == 0) {
+    // Register ft0.64 has encoding 34 with subsequent encodings interleaved
+    // with 32 bit floating point registers. See riscv.h
```

Comment:
is that `riscv.h` from Capstone? If so, expand on the file name/directory to make it more clear